wrappers/src/shared_realm_cs.cpp

```diff
@@ -460,13 +460,44 @@ REALM_EXPORT void thread_safe_reference_destroy(ThreadSafeReference* reference)
     delete reference;
 }
 
-REALM_EXPORT void shared_realm_write_copy(SharedRealm* realm, uint16_t* path, size_t path_len, char* encryption_key, NativeException::Marshallable& ex)
+REALM_EXPORT void shared_realm_export_to_local(SharedRealm* realm, Configuration configuration, uint8_t* encryption_key, NativeException::Marshallable& ex)
+{
+    handle_errors(ex, [&]() {
+        Realm::Config config;
+        config.path = Utf16StringAccessor(configuration.path, configuration.path_len);
+        if (encryption_key) {
+            auto& key = *reinterpret_cast<std::array<char, 64>*>(encryption_key);
+            config.encryption_key = std::vector<char>(key.begin(), key.end());
+        }
+
+        realm->get()->export_to(std::move(config));
+    });
+}
+
+REALM_EXPORT void shared_realm_export_to_sync(SharedRealm* realm, Configuration configuration, SyncConfiguration sync_configuration, SchemaObject* objects, int objects_length, SchemaProperty* properties, uint8_t* encryption_key, NativeException::Marshallable& ex)
```

Comment:
We don't need to pass the schema to this method as it's unlikely to be used in the copy. We can unify the `export_to_local` and `export_to_sync` functions by treating the sync configuration as optional (since it's a struct, it can't be really optional, but we can check for presence of `User` or something).

Reply:
Done.